### ISSUE TITLE
No longer launches games on Android

### ISSUE THREAD

**Issue by scottywtf** [notaz/picodrive#160]
Using RetroArch core on a Retroid Pocket 3+ games that were once working have stopped as of a couple months ago.  Games launch to a black screen, and if restarted, you get a warning message “developed for use only with ntsc Genesis systems”. I have the region set to auto.  Again this was all working fine a couple months ago

**Comment by irixxxx** [comment]
Please be more concise:
- which games are giving you problems?
- is this problem also occurring with retroarch on other devices, or on a desktop system?
- have you checked older versions on https://git.libretro.com/libretro/picodrive/-/pipelines (only those having a different commit hash are relevant here)?

Please try narrowing the problem down a bit. As it is, I haven't noticed any problem with my own Android devices.

**Comment by irixxxx** [comment]
No, I don't think so. You can ignore all pipelines created by m4xw though.
